src/test/java/io/github/intellij/dlanguage/features/DFoldingBuilderTest.java

```diff
@@ -15,9 +15,9 @@ public DFoldingBuilderTest() {
 
     // Fold00001 should really result in "  ". OPENCOM and CLOSECOM remains in
     // the text, but the rest gets collapsed to "  ".
-    public void testFold00001() throws Throwable {
-        doTest(/*"main (Function) */"{...}");
-    }
+//    public void testFold00001() throws Throwable {
```

Comment:
For future reference:
you should be able to use the `@Ignore` annotation to ignore tests

Reply:
Got it, thanks